lib/beacon_web/live/admin/page_live/form_component.ex

```diff
@@ -39,7 +39,9 @@ defmodule BeaconWeb.Admin.PageLive.FormComponent do
   end
 
   defp save_page(socket, :new, page_params) do
-    case Pages.create_page(page_params) do
+    parsed_params = Map.new(page_params, fn {key, value} -> {String.to_existing_atom(key), value} end)
+
+    case Pages.create_page(parsed_params) do
```

Comment:
Elsewhere we create pages with atom-key params, so here we also convert to atom keys